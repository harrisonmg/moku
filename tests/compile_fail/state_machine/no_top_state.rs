use moku::*;

#[state_machine]
mod blinky {
    use moku::*;

    #[machine_module]
    mod state_machine {}
}

fn main() {}
