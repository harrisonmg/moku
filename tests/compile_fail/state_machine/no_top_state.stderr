error: no `moku::TopState` was defined in this module
 --> tests/compile_fail/state_machine/no_top_state.rs:4:1
  |
4 | mod blinky {
  | ^^^
